<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lineWindow</class>
 <widget class="QMainWindow" name="lineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>线路信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    border-radius: 12px;    /* 圆角半径 */
}
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
#centralwidget {
    border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
    border-radius: 12px;
}

#listWidget {
    border-image: url(:/images/mainwindow/mainwindow_background.png) 0 0 0 0 stretch stretch;
    border-bottom-right-radius: 12px;
	border-bottom-left-radius: 12px;
	border: none;
}
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleBar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
/* 关闭按钮 */
#closeButton { image:url(:/images/mainwindow/close_normal.png); }
#closeButton:hover { image:url(:/images/mainwindow/close_hover.png); }

/* 最小化按钮 */
#minButton { image:url(:/images/mainwindow/min_normal.png); }
#minButton:hover { image:url(:/images/mainwindow/min_hover.png); }

/* 最大化按钮 */
#maxButton { image:url(:/images/mainwindow/max_normal.png); }
#maxButton:hover { image:url(:/images/mainwindow/max_hover.png); }

/* 标题栏背景及圆角 */
#titleBar {
    border-image: url(:/images/mainwindow/bar_background.png) 0 0 0 0 stretch stretch;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
}

/* 按钮透明背景 */
#titleBar QToolButton {
    border: none;
    background: transparent;
}
       </string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="btnZone" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="closeButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="minButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="maxButton">
            <property name="minimumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>15</width>
              <height>15</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1279</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
           </property>
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="batchSize">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/* 上一页按钮 */
QPushButton#prevBtn {
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
    /* 如果图片正好是按钮大小，也可以直接用 image:url() */
	color: rgb(55,55,55);
	border-radius:12px; 
}
QPushButton#prevBtn:hover {
    border-image: url(:/images/sideBar/black3.png) 0 0 0 0 stretch stretch;
}

/* 下一页按钮 */
QPushButton#nextBtn {
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
	color: rgb(55,55,55);
	border-radius:12px; 
}
QPushButton#nextBtn:hover {
    border-image: url(:/images/sideBar/black3.png) 0 0 0 0 stretch stretch;
}


QPushButton#cancel {
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
	color: rgb(55,55,55);
	border-radius:12px; 
}
QPushButton#cancel:hover {
    border-image: url(:/images/sideBar/black3.png) 0 0 0 0 stretch stretch;
}


/* 输入框背景 */
QLineEdit#pageEdit {
    border: none;                      /* 去掉默认边框 */
    border-image: url(:/images/sideBar/black2.png) 0 0 0 0 stretch stretch;
	color: rgb(55,55,55);
	border-radius:12px; 
    /* 去掉原来的 padding-left */
    padding: 0;
    /* 让文字水平垂直都居中 */
    qproperty-alignment: AlignCenter;
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="cancel">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>取消选择</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="prevBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>上一页</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pageEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>下一页</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
